split.js

```diff
@@ -211,6 +211,9 @@ var Split = function (ids, options) {
     // | <- this.start                                        this.size -> |
     function drag (e) {
         var offset;
+        var a = elements[this.a].element;
+        var b = elements[this.b].element;
+        var isVertical = e.target.classList.contains("gutter-vertical");
```

Comment:
I think you can just pass `direction` instead of `isVertical`. It would be more consistent with the other callback signatures.